flytekit/annotated/reference.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import Dict, Type
+
+from flytekit.annotated.launch_plan import ReferenceLaunchPlan
+from flytekit.annotated.reference_task import ReferenceTask
+from flytekit.annotated.workflow import ReferenceWorkflow
+from flytekit.models.core import identifier as _identifier_model
+
+
+def get_reference_entity(
```

Comment:
is this just used for testing?

Reply:
no, this is exported at the flytekit layer.  do you not like it?  I thought it'd be helpful.  I should probably add a raise Exception though.

Reply:
i can delete it if you think it's not helpful.